crates/rpc/rpc/src/eth/core.rs

```diff
@@ -428,6 +437,20 @@ where
     pub const fn blocking_task_guard(&self) -> &BlockingTaskGuard {
         &self.blocking_task_guard
     }
+
+    /// Returns [`broadcast::Receiver`] of new raw transactions
+    #[inline]
+    pub fn subscribe_to_raw_transactions(&self) -> broadcast::Receiver<Bytes> {
+        self.raw_tx_sender.subscribe()
+    }
+
+    /// Broadcasts raw transaction if there are active subscribers.
+    #[inline]
+    pub fn broadcast_raw_transaction(&self, raw_tx: Bytes) {
+        if self.raw_tx_sender.send(raw_tx).is_err() {
+            trace!("no receivers for raw transaction");
+        }
```

Comment:
for almost 100% of the time this entire feature will be useless, so we can ignore the error here entirely because this will just be noise